splunk-otel-android/src/main/java/com/splunk/rum/RumAttributeAppender.java

```diff
@@ -39,6 +39,7 @@
     //note: these will be in the otel semantic conventions as of spec v1.6.0
     static final AttributeKey<String> NETWORK_TYPE_KEY = stringKey("host.connection.type");
     static final AttributeKey<String> NETWORK_SUBTYPE_KEY = stringKey("host.connection.subtype");
+    static final AttributeKey<String> SPLUNK_OPERATION_KEY = stringKey("_splunk_operation");
```

Comment:
I didn't see anything that would remove or filter out this attribute before export. Are we ok with it making its way into ingest?

Reply:
yeah... right now we'll get an extra attribute, which isn't harmful (aside from a bit of extra data). A further refinement would be to figure out how to elide this attribute in the encoder, but I think that would require writing the entire `Writer<Span>` implementation or doing some really dirty hacks. Because this ends up in a JSON array of tags, we'd have to be clever about removing commas conditionally, etc.